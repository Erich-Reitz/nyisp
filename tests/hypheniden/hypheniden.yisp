(set a_b 1)
a_b

(set c-b 2)
c-b