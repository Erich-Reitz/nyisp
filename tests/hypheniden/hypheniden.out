nil
1
nil
2
